definitions/checks/foreman/puppet_class_duplicates.rb

```diff
@@ -0,0 +1,45 @@
+module Checks
+  module Foreman
+    class PuppetClassDuplicates < ForemanMaintain::Check
+      metadata do
+        label :puppet_class_duplicates
+        for_feature :foreman_database
+        description 'Check for duplicate Puppet class records'
+        tags :pre_upgrade
```

Comment:
It would be good to wait for the core PR to be merged, and then add a confine block here to enable this only on versions < than the version that has the fix: once the unique constraint is there, ther is no point on running this check anymore

Reply:
it's already merged and present will be present in 1.20, does the confine block below address it?

Reply:
I think it actually could be `check_max_version('foreman', '1.19')`, as in `1.20` there would already be the index present